stdlib/testing/testdata/aggregate_empty_window.flux

```diff
@@ -41,18 +43,15 @@ outData = "
 ,,2,2018-05-22T19:53:26Z,2018-05-22T19:55:00Z,2018-05-22T19:54:00Z,usage_idle,cpu,cpu-total,host.local,271.637327705587
 ,,2,2018-05-22T19:53:26Z,2018-05-22T19:55:00Z,2018-05-22T19:54:30Z,usage_idle,cpu,cpu-total,host.local,156.19056189225455
 ,,2,2018-05-22T19:53:26Z,2018-05-22T19:55:00Z,2018-05-22T19:55:00Z,usage_idle,cpu,cpu-total,host.local,
-"
-
-
-
-t_window_generate_empty = (table=<-) =>
-  table
-	|> range(start:2018-05-22T19:53:26Z, stop: 2018-05-22T19:55:00Z)
-	|> aggregateWindow(every:30s,fn:sum)
-
-mem = testing.loadMem
-store = testing.loadStorage
-
-want = mem(csv: outData)
-got = store(csv: inData) |> t_window_generate_empty()
-testing.assertEquals(name: "aggregate_empty_window", want:want, got: got)
\ No newline at end of file
+"
+t_window_generate_empty = (table=<-) =>
+	(table
+		|> range(start: 2018-05-22T19:53:26Z, stop: 2018-05-22T19:55:00Z)
+		|> aggregateWindow(every: 30s, fn: sum))
+mem = testing.loadMem
+store = testing.loadStorage
+want = mem(csv: outData)
+got = store(csv: inData)
+	|> t_window_generate_empty()
+
+testing.assertEquals(name: "aggregate_empty_window", want: want, got: got)
```

Comment:
why aren't these using the new testing framework?

Reply:
The script skipped these. I'll update.